### DISCUSSION TITLE
🤔 [QUESTION] 能否提供按需加载的解决方案？

### DISCUSSION THREAD

**Discussion by blckrabbit** [antvis/G2Plot#3583] [Q&A] [answered]
### 💻 Link to minimal reproduction

Please provide a link by forking these links [g2plot](https://codesandbox.io/s/g2plot-reproduction-template-gyej1) or GitHub repo. What is a minimal reproduction, and why is it required?

### 🚑 Any additional [like screenshots]

* **G2Plot Version**:2.4.20
* **Platform**:
能否提供按需加载的解决方案？

**Comment by hustcc** [comment, marked as answer]
G2Plot 层提供按需引入，带来的收益并不多，顶多优化 10~20% 的包大小。
